Components/Css/homepage.css

```diff
@@ -0,0 +1,29 @@
+.home-images {
+    height: 22rem;
+    width: 22rem;
+    margin-top: 1rem;
+}
+
+.home-images img {
+    height: 22rem;
+}
+
+.categories {
+    display: flex;
+    justify-content: space-evenly;
```

Comment:
-you can made two containers and then give spaces......like logo at left....other header icons on left